profiling/src/allocation/allocation_le83.rs

```diff
@@ -374,12 +377,13 @@ unsafe fn alloc_prof_orig_alloc(len: size_t) -> *mut c_void {
 
 /// This function exists because when calling `zend_mm_set_custom_handlers()`,
 /// you need to pass a pointer to a `free()` function as well, otherwise your
-/// custom handlers won't be installed. We can not just point to the original
+/// custom handlers won't be installed. We cannot just point to the original
 /// `zend::_zend_mm_free()` as the function definitions differ.
 unsafe extern "C" fn alloc_prof_free(ptr: *mut c_void) {
     tls_zend_mm_state_get!(free)(ptr);
 }
 
+#[export_name = "ddog_php_prof_alloc_prof_prev_free"]
```

Comment:
```suggestion
```

Do we need this and if so, why do we not need to change the exported symbol name for the other functions?

Reply:
No, I don't even remember why I put it on there, it was during experimentation with some error I was getting. Removed.